crates/uv/tests/it/python_install.rs

```diff
@@ -660,6 +660,79 @@ fn python_install_freethreaded() {
     "###);
 }
 
+#[test]
+fn python_install_debug() {
+    let context: TestContext = TestContext::new_with_versions(&[])
+        .with_filtered_python_keys()
+        .with_filtered_exe_suffix()
+        .with_managed_python_dirs();
+
+    // Install the latest version
+    uv_snapshot!(context.filters(), context.python_install().arg("--preview").arg("3.13d"), @r"
+    success: false
+    exit_code: 2
+    ----- stdout -----
+
+    ----- stderr -----
+    error: No download found for request: cpython-3.13d-[PLATFORM]
```

Comment:
I think something is wrong with `generate-download-metadata.py` because this download should exist.

Reply:
I just did some debugging here, it looks like the regex in `_filename_re` is picking up `-debug` as being a 4th piece of the triple, e.g. `aarch64-apple-darwin-debug`

So then the build options / variant processing is not working correctly.

This happens to not be a problem for the free-threaded builds because they are all `freethreaded+pgo` or `freethreaded+pgo+lto`, and `+` is not allowed as a character in the triple portion of the regex.

Reply:
> I just did some debugging here, it looks like the regex in `_filename_re` is picking up `-debug` as being a 4th piece of the triple, e.g. `aarch64-apple-darwin-debug`

Negative lookahead may be a solution?